RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -144,4 +144,57 @@ extension BlockingObservable {
         
         return element
     }
-}
\ No newline at end of file
+}
+
+extension BlockingObservable {
+    /**
+     Blocks current thread until sequence terminates.
+     
+     If sequence terminates with error before producing first element, terminating error will be thrown.
+     
+     - returns: Returns the only element of an sequence, and reports an exception if there is not exactly one element in the observable sequence.
+     */
```

Comment:
I think it would probably be better to say `error` then `exception` in context of Swift. But I can change that after merging. Not a big deal.